app/Platform/AppServiceProvider.php

```diff
@@ -108,6 +110,7 @@ public function boot(): void
             $schedule = $this->app->make(Schedule::class);
 
             $schedule->command(UpdateAwardsStaticData::class)->everyMinute();
+            $schedule->command(UpdateTotalGamesCount::class)->everyMinute();
```

Comment:
This is probably a fast operation, but does it need to run every minute? It should probably just run in response to the `AchievementPublished` and `AchievementUnpublished` events.

Reply:
Yes, I agree. Updated in latest.

However, this has surfaced what I believe is a high-severity regression that I believe we should hotfix in prod as soon as possible.

The regression is in `updateAchievementFlag()` and `updateAchievementType()`.

Here's the current implementation of `updateAchievementFlag()`:
```php
function updateAchievementFlag(int|string|array $inputAchievementIds, int $newFlag): void
{
    $achievementIds = is_array($inputAchievementIds) ? $inputAchievementIds : [$inputAchievementIds];

    Achievement::whereIn('ID', $achievementIds)
        ->where('Flags', '!=', $newFlag)
        ->update(['Flags' => $newFlag]);
}
```

`->update()` in a query builder instance is directly updating the database records without triggering Eloquent events like `updated`. In other words, if someone is using the "Manage Core Achievements" page to promote/demote something, those `AchievementPublished` and `AchievementUnpublished` jobs aren't being dispatched. Quite a gotcha.